### DISCUSSION TITLE
Start minimized and/or start on boot?

### DISCUSSION THREAD

**Discussion by tcconway** [cpvalente/ontime#1319] [Discussion]
Is there a way to set the server app to start on boot and make it minimized? Thanks.

**Comment by cpvalente** [comment]
> Is there a way to set the server app to start on boot and make it minimized? Thanks.

Hi @tcconway , this sounds like something you would need to manage with your OS. But I can try to help
What are you calling the server app? Is this the desktop app or a headless install like docker?

What OS are you using?

**Comment by tcconway** [comment]
OS: Windows 11

So, it looks like OnTime is not adhering to when you set the shortcut to start as minimized.

Steps to repeat (on Windows 11):
- Right-click on the shortcut that will start OnTime and choose Properties
- In the "Run" dropdown, choose "Minimized"
- Double click the shortcut

**Reply by cpvalente** [reply]
Thank you for clarifying, I had a quick look around and this seems to be something we need to manage manually
I will see if I find a good reference, but I would expect that we wont be able to get to this until after new years as this is a busy season for all of us here
